packages/@storefront/core/src/core/url-beautifier/details/parser.ts

```diff
@@ -18,6 +18,13 @@ export class DetailsUrlParser extends UrlParser<UrlBeautifier.DetailsUrlState> {
     const title = path.shift();
     const id = path.pop();
     let variants = [];
+    const queryParams: {[key: string]: string} = {};
+
+    if (Object.keys(uri.query).length) {
+      const filteredQueryParams = DetailsUrlParser.extractFilteredQueryParams(uri.query);
```

Comment:
This function feels a bit redundant. Is there a reason that we don't extract the `area` and `collection` properties directly from the `uri.query` object?

Reply:
That makes sense, will remove unnecessary function.